### ISSUE TITLE
[6.*] "npm run hot" fails with "Error: Cannot find module 'webpack-cli/bin/config-yargs'"

### ISSUE THREAD

**Issue by rderimay** [JeffreyWay/laravel-mix#2584]
-   Laravel Mix Version: 6.0.0-beta.11 (`npm list --depth=0`)
-   Node Version (`node -v`): v15.0.1
-   NPM Version (`npm -v`): 7.0.3
-   OS: MacOS

### Description:
`npm run hot` fails with "Error: Cannot find module 'webpack-cli/bin/config-yargs'" 

```
Error: Cannot find module 'webpack-cli/bin/config-yargs'
Require stack:
- /Users/.../node_modules/webpack-dev-server/bin/webpack-dev-server.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:900:15)
    at Function.Module._load (node:internal/modules/cjs/loader:745:27)
    at Module.require (node:internal/modules/cjs/loader:972:19)
    at require (node:internal/modules/cjs/helpers:88:18)
    at Object.<anonymous> (/Users/.../node_modules/webpack-dev-server/bin/webpack-dev-server.js:65:1)
    at Module._compile (node:internal/modules/cjs/loader:1083:30)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1112:10)
    at Module.load (node:internal/modules/cjs/loader:948:32)
    at Function.Module._load (node:internal/modules/cjs/loader:789:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:72:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/.../node_modules/webpack-dev-server/bin/webpack-dev-server.js'
  ]
}
```

### Steps To Reproduce:
On a fresh install with folowing package.json, run `npm run hot` (`npm run dev` works like a charm)

```
"devDependencies": {
        "@babel/core": "^7.12.3",
        "@sentry/browser": "^5.27.1",
        "@sentry/integrations": "^5.27.1",
        "@sentry/webpack-plugin": "^1.13.0",
        "@vue/component-compiler-utils": "^3.2.0",
        "axios": "^0.20",
        "babel-eslint": "^10.1.0",
        "babel-generator": "^6.26.1",
        "babel-helpers": "^6.24.1",
        "babel-plugin-lodash": "^3.3.4",
        "babel-plugin-syntax-dynamic-import": "^6.18.0",
        "babel-plugin-transform-class-properties": "^6.24.1",
        "babel-plugin-transform-object-rest-spread": "^6.26.0",
        "babel-plugin-transform-runtime": "^6.23.0",
        "babel-preset-env": "^1.7.0",
        "babel-preset-stage-3": "^6.24.1",
        "babel-template": "^6.26.0",
        "babel-traverse": "^6.26.0",
        "cross-env": "^7.0",
        "dayjs": "^1.9.3",
        "eslint": "^7.11.0",
        "eslint-config-prettier": "^6.12.0",
        "eslint-plugin-compat": "^3.8.0",
        "eslint-plugin-prettier": "^3.1.4",
        "eslint-plugin-vue": "^7.0.1",
        "laravel-mix": "^6.0.0-beta.11",
        "resolve-url-loader": "^3.1.0",
        "typeface-inconsolata": "^1.1.13",
        "typeface-pt-sans": "^1.1.13",
        "typeface-pt-sans-narrow": "^1.1.13",
        "vue": "^2.6.12",
        "vue-analytics": "^5.22.1",
        "vue-i18n": "^8.22.1",
        "vue-meta": "^2.4.0",
        "vue-router": "^3.4.8",
        "vue-template-compiler": "^2.6.12",
        "vuex": "^3.5.1",
        "vuex-persistedstate": "^2.7.1",
        "vuex-router-sync": "^5.0.0",
        "webpack": "^5.2.0",
        "webpack-bundle-analyzer": "^3.9.0",
        "webpackbar": "^4.0.0"
    }
```

**Comment by thecrypticace** [comment]
I'm unable to reproduce this with the provided package.json. Could you please provide a full reproduction repo?

**Comment by ankurk91** [comment]
https://github.com/webpack/webpack-dev-server/issues/2424#issuecomment-581319104

**Comment by thecrypticace** [comment]
Yep, I've seen that discussion. I went through it after seeing your issue and verified we use `webpack serve` for hot module replacement as required with webpack 5. I created a small test project and was unable to reproduce the error you saw above.

Have you by chance not updated your npm script for `hot`? (See: https://github.com/JeffreyWay/laravel-mix/blob/master/UPGRADE.md#update-your-npm-scripts)
